lms/static/sass/views/_homepage.scss

```diff
@@ -49,7 +50,7 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
 
         img {
           width: 100%;
-          height: auto;
+          min-height: $course-image-height;
```

Comment:
I believe this line still needs to be reverted now that height is'n't being adjusted. (related to responsive cases)